src/views/core.clj

```diff
@@ -2,10 +2,11 @@
   (:import
     [java.util.concurrent ArrayBlockingQueue TimeUnit])
   (:require
-    [views.protocols :refer [IView id data relevant?]]
-    [plumbing.core :refer [swap-pair!]]
     [clojure.tools.logging :refer [info debug error]]
-    [environ.core :refer [env]]))
+    [environ.core :refer [env]]
+    [plumbing.core :refer [swap-pair!]]
+    [views.hash :refer [hash]]
```

Comment:
Do we not need to exclude the core hash function using (:refer-clojure :exclude [hash])?